splinterd/src/config/default.rs

```diff
@@ -79,8 +79,8 @@ impl DefaultConfig {
             bind: Some(String::from("127.0.0.1:8080")),
             #[cfg(feature = "database")]
             database: Some(String::from("127.0.0.1:5432")),
-            registry_backend: None,
-            registry_file: None,
+            registry_backend: Some(String::from("FILE")),
+            registry_file: Some(String::from("/etc/splinter/")),
```

Comment:
Is this option expecting a filename or a directory?

Reply:
It's expecting a filename, but it's passed in as an option to create the splinter daemon, so I figured this would ensure the default would pass along a None option to that if the registry_file isn't a file.

Reply:
This should include the filename in that case.